src/main/antlr4/Manifold.g4

```diff
@@ -89,14 +92,23 @@ lvalue:
   ;
 
 // TODO: declarations as expressions
-expression: /* declaration | */ rvalue;
+expression:
+  rvalue
+  // | declaration
+  ;
+
+statement:
+    expression #ExpressionStatment
+    | 'import' STRING_VALUE #ImportStatement
+    ;
+
+STATEMENT_TERMINATOR: ';';
```

Comment:
This language is intended to be single assignment funcitonal-like. I strongly urge you to consider sticking with the everything-is-an-expression paradigm, as all modern functional programming languages do.